frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -686,6 +686,16 @@ To:
             UIManager:askForRestart()
         end,
     })
+    table.insert(self.menu_items.developer_options.sub_item_table, {
+        text = _("Anti-alias UI elements"),
```

Comment:
Please move this out of just in-between the somehow related xtext & rtl mirrorring :) ie. above near the "C blitter" thing, to which "antialias" sounds more related.
(I let @NiLuJe tell if it's worth having this toggable.)

Reply:
It looks to be fairly non-intrusive, so, eh, why not ;).

I'd make the label more precise though, because "UI elements" is kind of an overly optimistic generalization, given that it really only affects rounder corners ;p.

Reply:
That strikes me as redundant (i.e., what other rounded corners would we even know are such a thing as rounded corners other than those we very explicitly know about because we draw them), but if you prefer. /shrug